bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -81,7 +81,7 @@ def __init__(self,
         if self.pre_model.config.pad_token_id is None and self.pre_model.config.eos_token_id is not None:
             self.pre_model.config.pad_token_id = self.pre_model.config.eos_token_id
 
-        self.tokenizer = prep_tokenizer(self.tokenizer)
+        self.tokenizer = prep_tokenizer(self.tokenizer, self.std_tokenizer)
```

Comment:
Why do we need both tokenizers to prep the tokenizer?

Reply:
The second tokenizer is optional, and when provided it caches tokenizer.std_token_phrases once for future use in TextCausalLMNext to immediately call up the precalculated std_token_phrases for each server token.